src/widget/wtrackmenu.cpp

```diff
@@ -2808,9 +2809,34 @@ void WTrackMenu::addToAutoDJ(PlaylistDAO::AutoDJSendLoc loc) {
         return;
     }
 
+    // If the "Replace" action was clicked and there are tracks in the
+    // AutoDJ queue, ask for confirmation.
     PlaylistDAO& playlistDao = m_pLibrary->trackCollectionManager()
                                        ->internalCollection()
                                        ->getPlaylistDAO();
+    const auto tracks = playlistDao.getAutoDJTrackIds();
```

Comment:
isn't this the same as the `trackIds` above?

Reply:
No, those are the tracks that are currently in the Auto DJ queue.
No tracks, no need to confirm replace.

I renamed it to `autoDjTracks`.